modules/core/src/main/java/org/apache/ignite/internal/managers/communication/GridIoManager.java

```diff
@@ -2835,7 +2835,11 @@ private void receiveFromChannel(
                 boolean exit = in.readBoolean();
 
                 if (exit) {
-                    rcvCtxs.remove(topic);
+                    ReceiverContext rcv0 = rcvCtxs.remove(topic);
```

Comment:
Why not name this variable simply "rcv" (instead of rcv0)?

Reply:
fixed